packages/js/src/dashboard/services/remote-data-provider.js

```diff
@@ -25,9 +25,15 @@ export class RemoteDataProvider {
 	getUrl( endpoint, params ) {
 		const url = new URL( endpoint );
 
-		for ( const [ key, value ] of Object.entries( params ) ) {
-			url.searchParams.append( key, value );
-		}
+		forEach( params, ( value, key ) => {
+			if ( typeof value === "object" ) {
```

Comment:
Bit weird you do import `forEach` and replace the existing entries version. But then continue to not use `isObject` 😉